src/test/ui/consts/fn_trait_refs.rs

```diff
@@ -68,13 +68,9 @@ const fn main() {
         2
     };
 
-    // FIXME(const_cmp_tuple)
     let test_one = test_fn(one);
-    assert!(test_one.0 == 1);
-    assert!(test_one.1 == 1);
-    assert!(test_one.2 == 1);
+    assert!(test_one == (1, 1, 1));
 
     let test_two = test_fn_mut(two);
-    assert!(test_two.0 == 1);
-    assert!(test_two.1 == 1);
+    assert!(test_two == (2, 2));
```

Comment:
These asserts don't work unless you put them in a const item, `const _: () = {}`

Reply:
They are in const fn main()

Reply:
they don't work that way

Reply:
ahh right that only checks that i builds